README.md

```diff
@@ -23,8 +23,16 @@ Cross platform desktop app, wrapping the [wire-webapp](https://github.com/wireap
 ### Prerequisites
 
 - [Node.js](https://nodejs.org/) >= 10
+- Npm
 - Git
+- Yarn (Install using the official instructions at https://yarnpkg.com/lang/en/docs/install/, and not using the package recommended by apt-get)
 
+### Install Dependencies
+
+```shell
+sudo apt install git npm nodejs
```

Comment:
I'm not really happy with this line for two reasons:

- it covers only the installation process on Linux
- there is no npm on apt-get (and it's not needed, as stated above)